app/scripts/components/exploration/atoms/selectedFor.ts

```diff
@@ -0,0 +1,12 @@
+import { atomWithUrlValueStability } from '$utils/params-location-atom/atom-with-url-value-stability';
+
+const hydrateSelectedForEditing = (value) => value === 'true';
+
+const dehydrateSelectedForEditing = (value) => value ? 'true' : 'false';
+
+export const selectedForEditingAtom = atomWithUrlValueStability({
+  initialValue: hydrateSelectedForEditing(new URLSearchParams(window.location.search).get('selectedForEditing')),
+  urlParam: 'selectedForEditing',
+  hydrate: hydrateSelectedForEditing,
+  dehydrate: dehydrateSelectedForEditing
+});
```

Comment:
⛏️ : This is a nit but can we change the file name to `selectedForEditing` or something more information to what this file is for?